csp/templatetags/csp.py

```diff
@@ -19,10 +19,9 @@ def __init__(self, nodelist):
 
     def render(self, context):
         output = self.nodelist.render(context)
-        nonce = context.get('request')
+        request = context.get('request')
+        nonce = request.csp_nonce() if hasattr(request, 'csp_nonce') else ''
```

Comment:
Oh man I'm a goofball!

I'm sitting here looking for a _specifically_ defined method and completely ignoring the partial attached to `request.csp_nonce`.

My apologies for the time sink. :)